docs/Types.puml

```diff
@@ -46,17 +52,27 @@ class Database<T> {
 
 class CrudoDb {
   Record<string, Database> databases
+  Record<string, StoreApi> storeApis
 
-  addDatabase(db: Database<T>): void
   gets<T>(schemaKey: string): Promise<T[]>
-  get<T>(id: Key, schemaKey: string): Promise<T|undefined>
-  create<T>(obj: T, schemaKey: string): Promise<T|undefined>
-  update<T>(obj: T, schemaKey: string): Promise<T|undefined>
-  delete<T>(obj: T, schemaKey: string): Promise<boolean>
+  get<T>(schemaKey: string, id: Key): Promise<T|undefined>
+  create<T>(schemaKey: string, obj: T): Promise<T|undefined>
+  update<T>(schemaKey: string, obj: T): Promise<T|undefined>
+  delete<T>(schemaKey: string, obj: T): Promise<boolean>
```

Comment:
One could argue, that returning a boolean doesn't make much sense. The promise gets either resolved (successful) or rejected (error case)